v3.0/reference/calicoctl/resources/networkpolicy.md

```diff
@@ -6,13 +6,13 @@ no_canonical: true
 A network policy resource (`NetworkPolicy`) represents an ordered set of rules which are applied
 to a collection of endpoints that match a [label selector](#selector).
 
-NetworkPolicy is a namespaced resource. NetworkPolicy in a specific namespace
-only applies to [WorkloadEndpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint)
+`NetworkPolicy` is a namespaced resource. `NetworkPolicy` in a specific namespace
+only applies to [`WorkloadEndpoints`]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint)
 in that namespace. Two resources are in the same namespace if the namespace
 value is set the same on both.
-See [GlobalNetworkPolicy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/globalnetworkpolicy) for non-namespaced Network Policy.
+See [`GlobalNetworkPolicy`]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/globalnetworkpolicy) for non-namespaced network policy.
```

Comment:
does this render correctly on the site when you use `` for a link?

Reply:
It's a good point....the rendering is not good in this case. It's not obvious that it's a link. I've adjusted this.